pkgs/by-name/gg/gg-jj/package.nix

```diff
@@ -33,10 +34,23 @@ rustPlatform.buildRustPackage (finalAttrs: {
   cargoHash = "sha256-AdatJNDqIoRHfaf81iFhOs2JGLIxy7agFJj96bFPj00=";
 
   npmDeps = fetchNpmDeps {
-    inherit (finalAttrs) pname version src;
-    hash = "sha256-izCl3pE15ocEGYOYCUR1iTR+82nDB06Ed4YOGRGByfI=";
+    inherit (finalAttrs)
+      pname
+      version
+      src
+      patches
+      ;
+    hash = "sha256-ehXGLpCCN+BNqtwjEatcfR0kQHj5WOofTDR5mLSVW0U=";
   };
 
+  patches = [
+    # Remove after https://github.com/gulbanana/gg/pull/68 is released
+    (fetchpatch {
+      url = "https://github.com/gulbanana/gg/commit/627de4bd148229a1feb327262b9f0b2a75077ed3.patch";
+      hash = "sha256-V7mGU8a4o1WVRpBzdAEvXhkvMGLmgrMi77fou0nvYFg=";
+    })
```

Comment:
Since it's not upstream yet, the patch should be remain vendored per pkgs/README.md.